tests/test_pagination.py

```diff
@@ -218,21 +217,59 @@ def test_study_filter(self, client, participants,
         ('/cavatica-task-genomic-files')
     ])
     def test_non_exist_study_filter(self, client, participants,
-                                    endpoint, study_id):
+                                    endpoint):
         """
         Test pagination of resources with a study filter that doesn't exist or
         is invalid
 
         Should return no results
         """
-        endpoint = '{}?study_id={}'.format(endpoint, study_id)
+        endpoint = '{}?study_id={}'.format(endpoint, 'SD_00000000')
         resp = client.get(endpoint)
         resp = json.loads(resp.data.decode('utf-8'))
 
         assert len(resp['results']) == 0
         assert resp['limit'] == 10
         assert resp['total'] == 0
 
+    @pytest.mark.parametrize('study_id', ['blah', 3489, 'PT_00001111'])
+    @pytest.mark.parametrize('endpoint', [
+        ('/participants'),
```

Comment:
Can this be replaced with `ENDPOINTS` from conftest?

Reply:
👍